src/borg/constants.py

```diff
@@ -1,21 +1,76 @@
 # this set must be kept complete, otherwise the RobustUnpacker might malfunction:
-ITEM_KEYS = frozenset(['path', 'source', 'rdev', 'chunks', 'chunks_healthy', 'hardlink_master',
-                       'mode', 'user', 'group', 'uid', 'gid', 'mtime', 'atime', 'ctime', 'birthtime', 'size',
-                       'xattrs', 'bsdflags', 'acl_nfs4', 'acl_access', 'acl_default', 'acl_extended',
-                       'part'])
+ITEM_KEYS = frozenset(
+    [
+        "path",
+        "source",
+        "rdev",
+        "chunks",
+        "chunks_healthy",
+        "hardlink_master",
+        "mode",
+        "user",
+        "group",
+        "uid",
+        "gid",
+        "mtime",
+        "atime",
+        "ctime",
+        "birthtime",
+        "size",
+        "xattrs",
+        "bsdflags",
+        "acl_nfs4",
+        "acl_access",
+        "acl_default",
+        "acl_extended",
+        "part",
+    ]
```

Comment:
strange. a lot of vspace usage is gone, but here another extreme one is still present.

Reply:
Not strange: black either puts each item on its own line or puts them all on the same line. This is too long to fit in 120 characters so black puts each item on a separate line.

black respects `# fmt: off/on` to suppress formatting in special situations. https://stackoverflow.com/a/58584557

Reply:
ah, that's nice to deal with a few cases.